fennel/lib/aggregate/aggregate.py

```diff
@@ -159,6 +159,22 @@ def signature(self):
         return f"lastk_{self.of}_{self.window.signature()}"
 
 
+class Stddev(AggregateType):
+    of: str
+
+    def to_proto(self):
```

Comment:
Could you add a test for to_proto in test_dataset.py ( it essentially tests if the proto generated is correct )